usagov/vote-gov-nvrf-app
# Feature/vote 730/maine prev address special case

https://cm-jira.usa.gov/browse/VOTE-730

Description: Implement a special case for "Previous Address" for Maine, based on the instructions in the PDF.

Test steps:

Case: User has registered to vote before (in any state)
1. Open the form locally. You will need to edit line 60 in index.html so that the form uses Maine.
2. Select the "New registration" path.
3. On the Address page, confirm the "I moved since I last registered.." checkbox is present. Select the checkbox.
4. Continue with filling out the filling out the fields until the final PDF.

Case: User has not registered to vote before (in any state)
1. Open the form locally. You will need to edit line 60 in index.html so that the form uses Maine.
2. Select the "New registration" path.
3. On the Address page, confirm the "I moved since I last registered.." checkbox is present. Do not select the checkbox.
4. Continue with filling out the filling out the fields until the final PDF.
5. Confirm the value "N/A" is filled out in the previous street address box.
